installers/npm/package-lock.json

```diff
@@ -1,301 +1,8 @@
 {
   "name": "@apollo/rover",
   "version": "0.0.10",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Unrelated: This file flaps a lot for me locally, fwiw.  In this case, it looks like some devs are using `npm@7` and some are using `npm@6`.  I recently setup a strict engines rule on https://github.com/apollographql/federation/pull/708 that tries to help make sure users aren't able to use the inverse npm version as their peers, which seems particularly useful during this more major time for npm version upgrades in the ecosystem (we waited _so_ long for npm 7!)